Tools/process_sensor_caldata.py

```diff
@@ -94,11 +94,11 @@ def is_valid_directory(parser, arg):
         if sensor_instance == 0:
             sensor_baro_0 = d.data
             print('found baro 0 data')
-        num_baros = 1
+	    num_baros = 1
```

Comment:
The script uses a mix of spaces and tabs. Can you please convert everything to spaces (which is prefered for python)?